.github/workflows/genai_lib.yml

```diff
@@ -0,0 +1,20 @@
+name: genai_lib
+on: pull_request
+jobs:
+  genai_lib:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Install OpenVINO
+        run: |
+          mkdir ./ov/
+          curl https://storage.openvinotoolkit.org/repositories/openvino/packages/nightly/2024.1.0-14645-e6dc0865128/l_openvino_toolkit_ubuntu20_2024.1.0.dev20240304_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
+          sudo ./ov/install_dependencies/install_openvino_dependencies.sh
+      - run: python -m pip install scikit-build
+      - run: source ./ov/setupvars.sh && python -m pip install .
+      - run: python -c "from openvino.genai.py_generate_pipeline import LLMPipeline"
```

Comment:
I wonder how it works, because `openvino_genai-2024.2.0.0/openvino/genai/libgenerate_pipeline_lib.so` does not contain `RPATH`:
```
$ readelf -d openvino_genai-2024.2.0.0/openvino/genai/libgenerate_pipeline_lib.so

Dynamic section at offset 0x546ff8 contains 29 entries:
  Tag        Type                         Name/Value
 0x0000000000000001 (NEEDED)             Shared library: [libopenvino.so.2410]
 0x0000000000000001 (NEEDED)             Shared library: [libstdc++.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libm.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libgcc_s.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
 0x000000000000000e (SONAME)             Library soname: [libgenerate_pipeline_lib.so]
 0x000000000000000c (INIT)               0x2f0000
 0x000000000000000d (FINI)               0x4baed8
 0x0000000000000019 (INIT_ARRAY)         0x536580
```

Reply:
UPDATE: as I can see, it does not work (as expected):
![image](https://github.com/pavel-esir/openvino.genai/assets/2566854/fe035d86-e046-4831-9bc0-d776b0d2d157)

Reply:
BTW, 
```
auditwheel repair openvino_genai-2024.2.0.0-cp310-cp310-linux_x86_64.whl --plat "manylinux_2_35_x86_64" --exclude libgenerate_pipeline_lib.so --exclude libopenvino.so.2410 --exclude libtbb.so.12.5
```
helps to get proper tags

Reply:
What are these tags and why would I care?